src/assets/data/edition/series/1/section/5/intro.json

```diff
@@ -531,23 +531,23 @@
 
-                        "<p class='no-indent spacebreak'>On the basis of our assessment of Webern’s handwriting, a further manuscript can also be assigned to this period: “Dies ist ein Lied” M 133 (<span class='link-todo'>version 2</span>, which is the basic layer in <strong>C</strong>). Since the autograph of “Ihr tratet zu dem Herde” M 142 ([<strong>C</strong>]) has not survived, it is naturally impossible to use these criteria to determine whether it was also written in 1909 or already in 1908.</p>",
+                        "<p class='no-indent spacebreak'>On the basis of our assessment of Webern’s handwriting, a further manuscript can also be assigned to this period: “Dies ist ein Lied” M 133 (<a (click)=\"ref.selectSvgSheet({complexId: 'op3', sheetId: 'M_133_TF2'})\">version 2</span>, which is the basic layer in <a (click)=\"ref.navigateToReportFragment({complexId: 'op3', fragmentId: 'source_C'})\"><strong>C</strong></a>). Since the autograph of “Ihr tratet zu dem Herde” M 142 ([<strong>C</strong>]) has not survived, it is naturally impossible to use these criteria to determine whether it was also written in 1909 or already in 1908.</p>",
```

Comment:
```suggestion
                        "<p class='no-indent spacebreak'>On the basis of our assessment of Webern’s handwriting, a further manuscript can also be assigned to this period: “Dies ist ein Lied” M 133 (<a (click)=\"ref.selectSvgSheet({complexId: 'op3', sheetId: 'M_133_TF2'})\">version 2</a>, which is the basic layer in <a (click)=\"ref.navigateToReportFragment({complexId: 'op3', fragmentId: 'source_C'})\"><strong>C</strong></a>). Since the autograph of “Ihr tratet zu dem Herde” M 142 ([<strong>C</strong>]) has not survived, it is naturally impossible to use these criteria to determine whether it was also written in 1909 or already in 1908.</p>",
```